python_src/src/lamp_py/performance_manager/l1_rt_metrics.py

```diff
@@ -161,10 +160,10 @@ def process_metrics_table(
                 ).over(
                     partition_by=(
                         trips_for_metrics.c.parent_station,
-                        trips_for_metrics.c.route_id,
+                        trips_for_metrics.c.branch_route_id,
                         trips_for_metrics.c.direction_id,
                     ),
-                    order_by=trips_for_metrics.c.sort_timestamp,
+                    order_by=trips_for_metrics.c.next_station_move,
```

Comment:
since we're subtracting `next_station_move` from `lag next_station_move` we can just sort on `next_station_move`